database/migrations/2023_01_17_162703_append_projects.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AppendProjects extends Migration
```

Comment:
The name of this class should be
```suggestion
class MakeReasonForDeletionNullableInProjectTable extends Migration
```
also update the file name according to it. make sure the file name should be in snake case only